CMakeLists.txt

```diff
@@ -113,7 +113,7 @@ set(f3d_strict_build_compile_options "")
 if(F3D_STRICT_BUILD)
   if(MSVC)
     # Warning C4275 is essentially noise, disabling it to silent an issue with jsoncpp library
-    set(f3d_strict_build_compile_options /W4 /WX /wd4275)
+    set(f3d_strict_build_compile_options /W4 /WX)
```

Comment:
how can reenabling a warning fix a warning ?

Reply:
Wait ^^  
I'm just running the CI right now.